pxr/imaging/plugin/hdRpr/renderDelegate.cpp

```diff
@@ -306,6 +307,14 @@ HdRenderSettingDescriptorList HdRprDelegate::GetRenderSettingDescriptors() const
     return m_settingDescriptors;
 }
 
+VtDictionary HdRprDelegate::GetRenderStats() const {
+    VtDictionary stats;
+    int numCompletedSamples = m_rprApiSharedPtr->GetNumCompletedSamples();
+    stats[HdPerfTokens->numCompletedSamples.GetString()] = numCompletedSamples;
+    stats[_tokens->percentDone.GetString()] = 100.0 * double(numCompletedSamples) / HdRprConfig::GetInstance().GetMaxSamples();
```

Comment:
In the blender plugin we do percent done = min ( samples/maxsamples, (num_pixels - active_pixels) / num_pixels)

Where active pixels is the one returned from adaptive sampling.